lib/cloud_controller/db.rb

```diff
@@ -42,6 +42,7 @@ def self.connect(opts, logger)
       add_connection_expiration_extension(db, opts)
       add_connection_validator_extension(db, opts)
       db.extension(:requires_unique_column_names_in_subquery)
+      connection_metrics_extension(db)
```

Comment:
**thought** As part of this we should make sure we update the scaling docs  https://docs.cloudfoundry.org/running/managing-cf/scaling-cloud-controller.html and the metrics doc https://docs.cloudfoundry.org/running/all_metrics.html#cc

Reply:
Good point! PR is open for the docs.